site/_posts/2017-01-16-destroy-the-monolith.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Last Week in ManageIQ: Destroy the Monolith"
+author: durandom
+date: 2017-01-16 09:24:51 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Moin](https://en.wikipedia.org/wiki/Moin), [durandom][https://github.com/durandom/] from the providers team here, 
+guiding you through the vast construction site of [ManageIQ](http://manageiq.org/).
+
+Last week we had [206 PRs][PRs merged last week] merged in manageiq core.
```

Comment:
I'm seeing 60 PRs merged when clicking this link.  Not sure if the number or the `[PRs merged last week]` filter is set up as expected.